src/containers/ServiceLocationUpdateForm.js

```diff
@@ -57,12 +59,19 @@ function ServiceLocationUpdateForm({ backend, serviceUser }) {
       return;
     }
 
-    backend.setLocalZip(fields.zipCode);
-    history.push(
-      routeWithParams(Routes.SERVICE_LOCATION_AVAILABLE, {
-        zip: fields.zipCode,
-      }),
-    );
+    backend
+      .saveServiceUser({ ...serviceUser.data, zip: fields.zipCode })
```

Comment:
is this going to create a brand new user? should we create an `updateServiceUser` instead?

Reply:
oh jk the function handles it